charts/incubator/privatebin/templates/common.yaml

```diff
@@ -0,0 +1,5 @@
+{{/* Make sure all variables are set properly */}}
+{{- include "tc.common.loader.init" . }}
+
+{{/* Render secrets for privatebin */}}
+{{- include "privatebin.secrets" . }}
```

Comment:
It's missing the  one more function. The apply. 
Was there any app that had it like that?